src/main/java/com/firstspringmvcapp/controllers/ApiController.java

```diff
@@ -0,0 +1,97 @@
+package com.firstspringmvcapp.controllers;
+
+import com.firstspringmvcapp.util.PersonNotFoundException;
+import jakarta.validation.Valid;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.*;
+import com.firstspringmvcapp.dto.PersonDTO;
+import com.firstspringmvcapp.models.Person;
+import com.firstspringmvcapp.security.JWTUtil;
+import com.firstspringmvcapp.services.PeopleService;
+import com.firstspringmvcapp.util.PersonErrorResponse;
+import com.firstspringmvcapp.util.PersonNotCreatedException;
+import com.firstspringmvcapp.util.PersonValidator;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class ApiController {
+
+    private final PeopleService peopleService;
+    private final PersonValidator personValidator;
+    private final ModelMapper modelMapper;
+    private final JWTUtil jwtUtil;
+
+    @Autowired
+    public ApiController(PeopleService peopleService, PersonValidator personValidator, ModelMapper modelMapper, JWTUtil jwtUtil) {
+        this.peopleService = peopleService;
+        this.personValidator = personValidator;
+        this.modelMapper = modelMapper;
+        this.jwtUtil = jwtUtil;
+    }
+
+    @GetMapping("/people")
+    public List<Person> getPeople() {
+        return peopleService.findAll();
+    }
+
+    @GetMapping("/people/{id}")
+    public Person getPerson(@PathVariable("id") int id) {
+        return peopleService.findOne(id);
+    }
+
+    @ExceptionHandler
+    private ResponseEntity<PersonErrorResponse> handleException(PersonNotFoundException e) {
```

Comment:
It's not pretty to define handlers in the controller class. Separate it in a class  https://www.baeldung.com/exception-handling-for-rest-with-spring#controlleradvice